docs/src/main/asciidoc/kubernetes.adoc

```diff
@@ -405,11 +403,11 @@ Below you will find tables describing all available types.
 
 .Docker 
 |====
-| Property | Type | Description | Default Value 
-| docker-file | String |   | Dockerfile 
-| registry | String |   |  |
-| auto-push-enabled | boolean |   | false 
-| auto-build-enabled | boolean |   | false 
+| Property           | Type    | Description | Default Value
+| docker-file        | String  |             | Dockerfile
+| registry           | String  |             |
+| auto-push-enabled  | boolean |             | false
+| auto-build-enabled | boolean |             | false
```

Comment:
Is it normal that you don't have the `docker.` prefix here, compared to `openshift.` and `s2i`?

Reply:
@iocanel it would be nice if you could settle this before the end of day, I'm wrapping up 1.1.1.Final.

Thanks!

Reply:
@gsmet: right! Just fixed that!